src/commentbot/verify_projectfile.jl

```diff
@@ -84,5 +87,5 @@ function verify_projectfile_from_sha(reponame, sha; auth=GitHub.AnonymousAuth())
         end
     end
 
-    return project, t.sha, projectfile_found, projectfile_valid, err
+    return project, sha, projectfile_found, projectfile_valid, err
```

Comment:
The returned `sha` is only correct for the `subdir` case. Otherwise the input commit `sha` is returned rather than the tree sha for the top directory.

Reply:
Ah, good catch! I fixed it here: https://github.com/JuliaRegistries/Registrator.jl/pull/287/commits/e39cc8da04270bbc73951af714c3162d68eccab9